lib/streamlit/constants.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022-2024)
```

Comment:
I wasn't sure where to put this file since `lib/streamlit/commands/experimental_query_params` seemed like a weird place and so did `lib/streamlit/runtime/state`. Happy to move whereever. For now, just put in `lib/streamlit`.